ffi/capi/js/examples/node/icu_capi_cdylib/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+# This file is part of ICU4X. For terms of use, please see the file
+# called LICENSE at the top level of the ICU4X source tree
+# (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+[package]
+name = "icu_capi_cdylib"
```

Comment:
Suggestion: rename all of these example-specific crates with the example name because it's confusing that we have multiple crates named `icu_capi_cdylib` and multiple files named `icu_capi_cdylib.wasm` with completely different origins. We can bikeshed the names if you like.